docs/guide-zh-CN/blocktypes.json

```diff
@@ -1,6 +0,0 @@
-{
-    "Warning:": "警告：",
-    "Note:": "注意：",
-    "Info:": "补充：",
-    "Tip:": "提示："
-}
```

Comment:
why is this file deleted?

Reply:
Sorry, that's my mistake, #16061 can be merged。